### PR TITLE
[12.0][MIG] mrp_multi_level

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
* You can know integrate with different forecasting mechanisms.
      As a consequence integration with stock_demand_estimate is moved
      to a new module (mrp_multi_level_estimate).
* As agreed by the authors, the module is re-licensed to LGPL-3.

@Eficent

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

File: mrp_multi_level/models/product_mrp_area.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+    main_supplier_id = fields.Many2one(
+        comodel_name='res.partner', string='Main Supplier',
+        compute='_compute_main_supplier', store=True,
+        index=True,
+    )
+    main_supplierinfo_id = fields.Many2one(
+        comodel_name='product.supplierinfo', string='Supplier Info',
+        compute='_compute_main_supplier', store=True,
+    )
+    supply_method = fields.Selection(
```
Comment (HviorForgeFlow) [inline mrp_multi_level/models/product_mrp_area.py:84]: 'move' action has been split in v12 to pull, push pull_push.
This is causing errors on menu product mrp area parameters.

```ValueError: Wrong value for product.mrp.area.supply_method: 'pull'```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]

File: mrp_multi_level/models/product_mrp_area.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply mrp_multi_level/models/product_mrp_area.py:84]: Thanks for noticing, fixed.

**Comment by LoisRForgeFlow** [comment]
@hveficent Could you update your review?

**Review by HviorForgeFlow — APPROVED** [review]
Code review
LGTM :+1:

**Comment by JordiBForgeFlow** [comment]
@lreficent can you check e5b2418 and backport to 11.0?

**Comment by LoisRForgeFlow** [comment]
/ocabot merge

EDIT: cancelled, pending to review last commit.

**Comment by LoisRForgeFlow** [comment]
@jbeficent Tested, it works good and only runs current company areas if you manually do it from the wizard. Proceeding to merge.

/ocabot merge

**Pull request closed without merging**